lib/television.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+// Define class
+const television = {
+
+  // Variables
+  powerOn: false,
+  volume: 10,
+  maxVol: 100,
+  minVol: 0,
+  maxChan: 500,
+  minChan: 0,
+  channel: 100,
+  mute: true,
+
+  // Methods
+  togglePower: function(){
+    this.powerOn = !this.powerOn;
```

Comment:
This works! It's a little more idiomatic to use a ternary for this though. The whole method can become one line.

```js
togglePower: function () {
  this.power === 'on' ? 'off' : 'on';
};
```